pkg/kind/kind.go

```diff
@@ -290,7 +290,14 @@ func recreateCluster(registry bool) error {
 }
 
 // createNewCluster creates a new Kind cluster
-func createNewCluster() error {
+func createNewCluster(extraMountHostPath string, extraMountContainerPath string) error {
+	extraMount := ""
+	if extraMountHostPath != "" && extraMountContainerPath != "" {
+		extraMount = fmt.Sprintf(`
+  extraMounts:
+  - hostPath: %s
+    containerPath: %s`, extraMountHostPath, extraMountContainerPath)
+	}
 
 	fmt.Println("☸ Creating Kind cluster...")
```

Comment:
Could we also log that we're creating extra mounts?  maybe just something as simple as `☸ Creating Kind cluster with extra mounts...` ?

Reply:
Good idea! I added the line if we are creating with `extraMounts`